### ISSUE TITLE
Button Refresh does'nt reload ReferenceManyField

### ISSUE THREAD

**Issue by sebdec** [marmelab/admin-on-rest#987]
On a show page, when I click on the refresh button, it doesn't refresh the referenceManyFields

I see an other issue which is merged, but it seems to do not resolve this problem, so I think it's a bug
https://github.com/marmelab/admin-on-rest/issues/279
https://github.com/marmelab/admin-on-rest/pull/290

**Comment by djhi** [comment]
Can you provide a repository or codepen reproducing the problem ?

**Comment by djhi** [comment]
Bump

**Comment by natrim** [comment]
seconding

example:
just follow tutorial and add `ReferenceManyField` to Edit from and click refresh
```
        <SimpleForm {...props}>
            <TextInput source="title" validate={[required]} />
            <LongTextInput source="body" />
            <ReferenceManyField label="User" source="userId" reference="users">
            <Datagrid>
                <TextField source="name" />
                </Datagrid>
            </ReferenceManyField>
        </SimpleForm>
```

when you use `ReferenceManyField` in some `Edit` or `Show` and click refresh
the `ReferenceManyField` is not refreshed

and it would propably be nice to be able refresh only `ReferenceManyField`
(ie. add some refresh icon button to label)

**Comment by fzaninotto** [comment]
Fixed by #1149
